### PR TITLE
[GAWB-1597] Switch position of name, namespace columns

### PR DESCRIPTION
**Pull request by aednichols**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs [hunk 1]
```diff
@@ -308,7 +308,7 @@
                   (item "synopsis")
                   (item "createDate")
                   (when (= :config (:type item))
-                    (mapv (get item "method" {}) ["namespace" "name" "snapshotId"]))])}]))
+                    (mapv (get item "method" {}) ["name" "namespace" "snapshotId"]))])}]))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs]: This isn't for the name and namespace columns, this is for the "Referenced Method" column.

**Review by MattPutnam — CHANGES_REQUESTED** [review]

File: src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs (hunk 1 above)
Comment (davidangb) [reply src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs]: agreed - I think you only need the changes to the `Table {:columns ...`

Comment (aednichols) [reply src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs]: Good call, saved us a bug:

screen shot 2017-04-05 at 10 18 10 am
vs.

screen shot 2017-04-05 at 10 19 22 am

**Review by davidangb — APPROVED** [review]
👍 code looks good; I did not test!

**Review by MattPutnam — APPROVED** [review]

**Pull request merged**